README.md

```diff
@@ -2,79 +2,74 @@
 
 > **General disclaimer:** Please be aware that this template is in public preview. Therefore, expect smaller bugs and issues when working with the solution. Please submit an Issue in GitHub if you come across any issues that you would like us to fix.
 
-# Description
-The Data Management template is, as the name suggessts, classified as a management function and is at the heart of the Enterprise Scale Analytics platform. It is responsible for the governance of the platform and enables communication to ingest data sources from Azure, third-party clouds and on-premises data sources.
+## Description
+
+The Data Management template is, as the name suggests, classified as a management function and is at the heart of the **Enterprise Scale Analytics** platform. It is responsible for the governance of the platform and enables communication to ingest data sources from Azure, third-party clouds and on-premises data sources.
```

Comment:
Please can we call out at the top a link to ESA Docs so that if people land on this they can go to docs first if they want.

Something like:

The Data Management template is, as the name suggests, classified as a management function and is at the heart of the Enterprise Scale Analytics platform. It is responsible for the governance of the platform and enables communication to ingest data sources from Azure, third-party clouds and on-premises data sources.

If you are want to understand the Enteprise Scale Analytics & AI Solution Pattern please refer to documention <docdocs link>